doc/index.rst

```diff
@@ -1,2 +1,12 @@
+Welcome to depfinder documentation!
+===================================
 
-.. include:: ../readme.rst
+.. include:: ../README.md
+   :parser: myst_parser.sphinx_
```

Comment:
ahhh this was one of the things that i was missing in my attempt

Reply:
Yep. If you include as toctree you don't need it but if you use `.. include::` then we need this.